test/flycheck-test.el

```diff
@@ -4283,6 +4283,13 @@ Why not:
    '(9 nil warning "printindex before document beginning: @printindex cp"
        :checker texinfo)))
 
+(flycheck-ert-def-checker-test typescript-tslint typescript nil
+                               (flycheck-ert-should-syntax-check
```

Comment:
Please fix the indentation here.  Look at how the other tests are indented.

To each Emacs the indentation rules for `flycheck-ert-def-checker-test`, do `M-x load-library RET flycheck-ret` before editing this file.